app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+class Post < ApplicationRecord
+  validates :content, presence: true, length: { maximum: 180 }
+  validates :user, presence: true
```

Comment:
You don't need to validate user presence here as your `belongs_to :user` association already takes care of this for you.